docs/articles_en/get_started/installing-openvino-overview/installing-openvino-linux-header/installing-openvino-zypper.rst

```diff
@@ -0,0 +1,205 @@
+.. {#openvino_docs_install_guides_installing_openvino_zypper}
+
+Install OpenVINO™ Runtime on Linux From ZYPPER Repository
+========================================================
+
+
+.. meta::
+   :description: Learn how to install OpenVINO™ Runtime on Linux operating
+                 system, using the ZYPPER repository.
+
+.. note::
+
+   Note that the ZYPPER distribution:
+
+   * offers both C/C++ and Python APIs
+   * does not offer support for NPU inference
+   * is dedicated to Linux users only
+   * additionally includes code samples
+
+.. tab-set::
+
+   .. tab-item:: System Requirements
+      :sync: system-requirements
+
+      | Full requirement listing is available in:
+      | :doc:`System Requirements Page <system_requirements>`
+
+      .. note::
+
+         OpenVINO RPM packages are compatible with and can be run on the following operating systems:
+
+         - RHEL 8.2 and higher
+         - Amazon Linux 2022 and 2023
+         - Rocky Linux 8.7, 8.8 and 9.2
+         - Alma Linux 8.7, 8.8 and 9.2
+         - Oracle Linux 8.7, 8.8 and 9.2
+         - Fedora 29 and higher up to 40
+         - OpenEuler 20.03 and 22.03
+         - Anolis OS 8.6 and 8.8
+         - CentOS Stream 8 and 9
```

Comment:
is it a correct list of OSes? who can I use https://download.opensuse.org/repositories/home:cabelo:intel/15.5/home:cabelo:intel.repo on Alma Linux?